.gitignore

```diff
@@ -26,6 +26,9 @@ cabal.project.local
 .stack-work/
 stack.yaml.lock
 
+# Tests
+big.csv
```

Comment:
Shouldn't it be `test/gen/big.csv`?

Reply:
I don't know :) Depends on whether wi'll have many big.csv files in different places :) 
Right now just big.csv without the path works fine, since we don't have any other big.csv files which we would rather commit. 
But I can specify the path of you think it's more appropriate.

Reply:
No need to specify, just wanted to confirm that it works 🙂